src/server/qgsserver.cpp

```diff
@@ -290,7 +290,7 @@ bool QgsServer::init()
 
 void QgsServer::putenv( const QString &var, const QString &val )
 {
-#ifdef _MSC_VER
+#ifdef Q_OS_WIN32
```

Comment:
this function should be using qputenv -- that's already got all the cross platform handling done for us!